environment.sh

```diff
@@ -1,17 +1,19 @@
 #!/bin/bash
 ############################################################
-# Help                                                     #
+# Help:                                      #
 ############################################################
 Help()
-{
+{ 
+  # Reset OPTIND so help can be invoked multiple times per shell session.
+  OPTIND=1
    # Display Help
-   echo "Add description of the script functions here."
+   echo "Script to automatically create and validate conda environments."
    echo
-   echo "Syntax: scriptTemplate [-h|t|v]"
+   echo "Syntax: source environment.sh [-h|t|v]"
    echo "options:"
    echo "h     Print this Help."
    echo "v     Verbose mode."
-   echo "t     Type of conda environment. Either simulation (default) or artifact."
+   echo "t     Type of conda environment. Either 'simulation' (default) or 'artifact'."
```

Comment:
This is just a ui thing, but I'd personally make the env type an argument (w/ a default of 'simulation') instead of an option. This would then be invoked like `source environmnent.sh -v` or `source enviornment.sh artifact -v`

This would also get rid of the weird situation where you're defining `env_type` as "simulation" on line 21 and then re-defining it in the event the user passes in something different via -t. `env_type` really isn't an option, it's a required arg.

Reply:
Yea @rmudambi and I went back and forth on this. Currently for a simulation environment you wouldn't have to provide any arguments which is really nice from a user perspective and that way there is no confusion as to what environment some specifies to build or use and what they are actually building using. I do think this is something getting input from RT would be helpful for because currently it might be not as obvious what environment someone is using or trying to activate.